docs/en-US/component/switch.md

```diff
@@ -79,37 +79,42 @@ switch/prevent-switching
+| inactive-text                | text displayed when in `off` state                                                                                                              | ^[string]                                        | ''      |
+| active-value                 | switch value when in `on` state                                                                                                                 | ^[boolean] / ^[string] / ^[number]               | true    |
+| inactive-value               | switch value when in `off` state                                                                                                                | ^[boolean] / ^[string] / ^[number]               | false   |
+| active-color ^(deprecated)   | background color when in `on` state ( deprecated, use CSS var `--el-switch-on-color` instead )                                                  | ^[string]                                        | ''      |
+| inactive-color ^(deprecated) | background color when in `off` state ( deprecated, use CSS var `--el-switch-off-color` instead )                                                | ^[string]                                        | ''      |
+| border-color ^(deprecated)   | border color of the switch ( deprecated, use CSS var `--el-switch-border-color` instead )                                                       | ^[string]                                        | ''      |
+| name                         | input name of Switch                                                                                                                            | ^[string]                                        | ''      |
+| validate-event               | whether to trigger form validation                                                                                                              | ^[boolean]                                       | true    |
+| before-change                | before-change hook before the switch state changes. If `false` is returned or a `Promise` is returned and then is rejected, will stop switching | ^[boolean] / ^[Function]`() => Promise<boolean>` | —       |
+| id                           | id for input                                                                                                                                    | ^[string]                                        | —       |
+| tabindex                     | tabindex for input                                                                                                                              | ^[string] / ^[number]                            | —       |
+| value ^(deprecated)          | binding value ( deprecated, use `model-value / v-model` instead )                                                                               | ^[boolean] / ^[string] / ^[number]               | false   |
```

Comment:
Why add deprecated attributes to the document?

Reply:
I found that the 'value' property is still included in the switch property declaration, so I added it. Let me remove it?

Reply:
I don't feel the need to add it to the docs.

Reply:
I agree, I have deleted it from the docs.